### DISCUSSION TITLE
How do you refresh warnings

### DISCUSSION THREAD

**Discussion by TinyTimothy2** [TabularEditor/TabularEditor3#1238] [🙋♀️TE3 Q&A] [answered]
How do you refresh warnings
There doesn't seem to be a button to refresh warnings (ie calculated state)
Some seem to go away if i can fix measure.
Otherwise I have to close and reopen.

**Comment by otykier** [comment, marked as answer]
Right now, warnings regarding object state (NoData / Calculation needed / etc.) are not automatically refresh. You'll need to close and reopen the model for those warnings to disappear. We're working on improving this.

Other types of warnings, like warnings within a DAX expression, should disappear automatically when the issue is fixed.
